.ci/azure-pipelines-package.yml

```diff
@@ -0,0 +1,131 @@
+jobs:
+- job: BuildPackage
+  displayName: 'Build Packages'
+
+  strategy:
+    matrix:
+      CentOS.amd64:
+        BuildConfiguration: centos.amd64
+      Fedora.amd64:
+        BuildConfiguration: fedora.amd64
+      Debian.amd64:
+        BuildConfiguration: debian.amd64
+      Debian.arm64:
+        BuildConfiguration: debian.arm64
+      Debian.armhf:
+        BuildConfiguration: debian.armhf
+      Ubuntu.amd64:
+        BuildConfiguration: ubuntu.amd64
+      Ubuntu.arm64:
+        BuildConfiguration: ubuntu.arm64
+      Ubuntu.armhf:
+        BuildConfiguration: ubuntu.armhf
+      Linux.amd64:
+        BuildConfiguration: linux.amd64
+      Windows.amd64:
+        BuildConfiguration: windows.amd64
```

Comment:
Why add the architecture if we only have one? MacOS and portable have none specified so it seems inconsistent.

Reply:
Legacy of having the 32-bit one too. I could remove it, but it would preclude adding that back in the future, or a Windows ARM, etc. Basically like Linux it's there for the flexibility to avoid having to add it again later.

Reply:
macOS might have an arm version after tonights' keynote

Reply:
See I *knew* I should have kept it! Hahaha.

Reply:
Anyways, this is something that can be addressed further as needs change in the future. This is purely cosmetic.